CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ Changes since v2.17
 
 ### Changes
 - The "Attachments (zip)" button in the UI now only downloads the current application attachments. Event attachments and previous versions of application attachments are left out. The full zip is still available via the API. (#2453)
+- Changes to theming: (#2588)
+  - Theme variables are now documented in [resources/config-defaults.edn](resources/config-defaults.edn).
+  - The theme variables `:danger-color` and `:phase-background-active` have been removed.
+  - The look of the default theme has changes a bit.
```

Comment:
Typo 
> has changed a bit.

But how? Should we document it in screenshots or text format? Just to say that it is rather insignificant change.

Reply:
I thought most people wouldn't care about the default theme that much. The screenshots are in this PR so I guess I could link to this PR?

Reply:
How about wording it 
> The default theme has minor visual changes.

That'd sound to me as unimportant enough.

Reply:
Or just list them there as text?